lms/static/js/student_account/views/FormView.js

```diff
@@ -172,7 +172,7 @@
 
                         if (key) {
                             test = this.validate(elements[i]);
-                            if (test.isValid) {
+                            if (test.isValid || $el.attr('id') === 'login-email') {
```

Comment:
Yes I'm disabling the error just for that specific element that has id ```login-email```. Which I think it's ok because there should only be one id per element.